### PR TITLE
Adding a few fingerprints to http and ssh

### PR DESCRIPTION
**Pull request by dheiland-r7**
## Description
Have several fingerprints iam adding one for Zyxel SSH. and three http_server for server header data returned from ssdp M-SEARCH request on UDP port 1900, included for Pelco camera, Netgear AP router and a generic response 

## Motivation and Context
Attempt to add some new IoT/embedded device fingerprints

## How Has This Been Tested?
Tested again lab device and validate with current recog scripts

## Types of changes
New fingerprints

### CODE REVIEW

**Review by jvoisin — APPROVED** [review]

File: xml/http_servers.xml [hunk 1]
```diff
@@ -4246,4 +4246,31 @@
     <param pos="0" name="service.cpe23" value="cpe:/a:acme:mini_httpd:-"/>
   </fingerprint>
 
-</fingerprints>
\ No newline at end of file
+  <fingerprint pattern="^Unspecified, UPnP/[\d\.]+, Unspecified$">
+    <description> UPNP server</description>
```
Comment (tsellers-r7) [inline xml/http_servers.xml:4250]: Remove the leading whitespace in the description.

File: xml/http_servers.xml [hunk 2]
```diff
@@ -4246,4 +4246,31 @@
...
+    <example>Unspecified, UPnP/1.0, Unspecified</example>
+    <param pos="0" name="service.family" value="UPnP"/>
+  </fingerprint>
+
+  <fingerprint pattern="^WNR2000v([0-9]) UPnP/[\d\.]+ miniupnpd/([\d\.]+)$">
+    <description>Netgear WNR2000v5 Router UPnP</description>
+    <example service.version="5" service.component.version="1.0">WNR2000v5 UPnP/1.0 miniupnpd/1.0</example>
+    <param pos="1" name="service.version"/>
```
Comment (tsellers-r7) [inline xml/http_servers.xml:4258]: Is this a `service.version` or `hw.version`?

File: xml/http_servers.xml [hunk 3]
```diff
@@ -4246,4 +4246,31 @@
...
-</fingerprints>
\ No newline at end of file
+  <fingerprint pattern="^Unspecified, UPnP/[\d\.]+, Unspecified$">
+    <description> UPNP server</description>
+    <example>Unspecified, UPnP/1.0, Unspecified</example>
+    <param pos="0" name="service.family" value="UPnP"/>
+  </fingerprint>
+
+  <fingerprint pattern="^WNR2000v([0-9]) UPnP/[\d\.]+ miniupnpd/([\d\.]+)$">
+    <description>Netgear WNR2000v5 Router UPnP</description>
```
Comment (tsellers-r7) [inline xml/http_servers.xml:4256]: Should `WNR2000` be `hw.product`?

File: xml/ssh_banners.xml [hunk 4]
```diff
@@ -2125,6 +2125,18 @@
     <param pos="0" name="service.product" value="WeOnlyDo SSH Server"/>
   </fingerprint>
 
+  <fingerprint pattern="^Zyxel SSH server$">
+    <description>Zyxel Firewall SSH service</description>
+    <example>Zyxel SSH server</example>
+    <param pos="0" name="service.vendor" value="Zyxel"/>
+    <param pos="0" name="service.family" value="Zywall"/>
+    <param pos="0" name="os.vendor" value="Zyxel"/>
+    <param pos="0" name="os.product" value="ZyNOS"/>
```
Comment (tsellers-r7) [inline xml/ssh_banners.xml:2134]: If you change this value to `ZyNOS Firmware` then `update_cpes.py` (https://github.com/rapid7/recog/blob/master/CONTRIBUTING.md#updating-cpes) will auto populate the CPE.

**Review by tsellers-r7 — CHANGES_REQUESTED** [review]

File: xml/http_servers.xml (hunk 3 above)
Comment (dheiland-r7) [reply xml/http_servers.xml:4256]: Yes, i would assume so. So i will add <param pos="0" name="hw.product" value="WNR2000"/> to it. Also should the hw_product.txt be updated with WNR2000?

Comment (tsellers-r7) [reply xml/http_servers.xml:4256]: `hw_product.txt`? If you mean the description, then yes please.

**Comment by hdm** [comment]
The hw_product.txt file is updated automatically when you run recog_standardize in bin, no need to do it by hand (and the diff is a good check for typos).

File: xml/http_servers.xml (hunk 3 above)
Comment (tsellers-r7) [reply xml/http_servers.xml:4256]: Doh, my brain was totally on the fields here. Thanks HD for the comment.

Comment (dheiland-r7) [reply xml/http_servers.xml:4256]: recog_standardize identified some issue on another fingerprint in  http_server.xml for "Proxmox api daemon" they used multiple cases so it wants to create multiple Proxmox and proxmox in the identifiers. I will correct that fingerprint to use same case and rerun recog_standardize -w and push my correctlion up shortly.

File: xml/http_servers.xml (hunk 1 above)
Comment (dheiland-r7) [reply xml/http_servers.xml]: completed

File: xml/http_servers.xml (hunk 2 above)
Comment (dheiland-r7) [reply xml/http_servers.xml]: switched to hw.version

File: xml/ssh_banners.xml (hunk 4 above)
Comment (dheiland-r7) [reply xml/ssh_banners.xml]: completed this change and ran update_cpes

**Comment by tsellers-r7** [comment]
Landed, thanks @dheiland-r7

**Pull request merged**